docs/maui/views/MediaElement.md

```diff
@@ -67,21 +67,104 @@ To access the `MediaElement` functionality, the following platform specific setu
+<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+<uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+<uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
+<uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK" />
+<uses-permission android:name="android.permission.MEDIA_CONTENT_CONTROL" />
+```
+
+#### Here is an example of required settings in `AndroidManifest.xml`
+
+```csharp
+<application android:allowBackup="true" android:icon="@mipmap/appicon" android:enableOnBackInvokedCallback="true" android:supportsRtl="true">
+<service android:name="CommunityToolkit.Maui.Media.Services" android:exported="false" android:enabled="true" android:foregroundServiceType="mediaPlayback">
+    <intent-filter>
+    <action android:name="android.intent.action.MEDIA_BUTTON" />
+    </intent-filter>
+    <intent-filter>
+    <action android:name="androidx.media3.session.MediaSessionService"/>
+    </intent-filter>
+</service>
+</application>
+<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+<uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+<uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
+<uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK" />
+<uses-permission android:name="android.permission.MEDIA_CONTENT_CONTROL" />
+```
+
+> [!NOTE]
+> This modification to the Android manifest enables metadata display when playing a video. It provides support for notifications and is essential
+ for notifications to function across all relevant APIs. The change introduces a service and grants necessary permissions.
+
 For a full example of this method included in an application please refer to the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Platforms/Android/MainActivity.cs)
 
-### [iOS/Mac Catalyst](#tab/macios)
+### [Mac Catalyst](#tab/mac)
 
-No setup is required.
+Edit the `Info.plist` for `MacCatalyst` and add the following keys.
+```csharp
+<key>NSAppTransportSecurity</key>
+<dict>
+    <key>NSAllowsArbitraryLoadsInMedia</key>
+    <true/>
+    <key>NSAllowsArbitraryLoads</key>
+    <true/>
+</dict>
+<key>UIBackgroundModes</key>
+<array>
+    <string>bluetooth-central</string>
+    <string>audio</string>
+</array>
+<key>NSLocalNetworkUsageDescription</key>
+<string></string>
+```
+
+### [iOS](#tab/ios)
+
+Edit the `Info.plist` for `iOS` and add the following keys.
+```csharp
+<key>NSAppTransportSecurity</key>
+<dict>
+    <key>NSAllowsArbitraryLoadsInMedia</key>
+    <true/>
+</dict>
```

Comment:
```suggestion
```

Unless there's a critical reason to keep this, let's remove `NSAppTransportSecurity` from the required setup because allowing arbitrary loads opens a large security vulnerability in the developer's app and it will cause the app to fail an App Store review. 

Apple strongly advises against `NSAllowsArbitraryLoads = true`:  https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloads

<img width="974" alt="image" src="https://github.com/MicrosoftDocs/CommunityToolkit/assets/13558917/8513c1ba-6c50-45a0-a03d-3ca7e48e153a">

Reply:
I'll start testing to see if this negatively impacts the display of lock screen controls. Will reply back in about an hour or so.

Reply:
Tested and verified working with removal of nstransport on iOS and Mac Catalyst. Only one minor issue. On ios simulator it does not work. But it does work on apple m2 mac mini, and on ipad 10 generation running iOS 17.2.